mobile/contacts.go

```diff
@@ -70,8 +72,17 @@ func (m *Mobile) ContactThreads(id string) (string, error) {
 }
 
 // SearchContacts calls core SearchContacts
-func (m *Mobile) SearchContacts(query *pb.ContactQuery, options *pb.QueryOptions, cb Callback) (func(), error) {
-	resCh, errCh, cancel, err := m.node.SearchContacts(query, options)
+func (m *Mobile) SearchContacts(query []byte, options []byte, cb Callback) (*CancelFn, error) {
```

Comment:
@andrewxhill I fixed this up here... turns out you can't just return `func()`, so I added this `CancelFn` object that will expose a single `call` method